cmd/root.go

```diff
@@ -56,15 +59,43 @@ var RootCmd = &cobra.Command{
 			panic(err)
 		}
 
-		currTime := time.Now().Unix()
+		var commuteTime int64
 		minute := 60
 		interval := int64(interval * minute)
 		var traveTime int64 // time leaving
 		var info directions.CommuteInfo
 		var shortest int = 0
 		var optimalTime string
 		var bestHour int
+		today := true
 		amPm := "AM"
+		commuteTime = time.Now().Unix()
+
+		if start != "" {
```

Comment:
All of this logic within the if statement should be its own _formatDate helper method in this file.

```go
if start != "" {
    commuteTime = _formatDateInput(start)
}
```